onnxscript/rewriter/ort_fusions/skip_normalization.py

```diff
@@ -218,10 +216,10 @@ def rewrite(
         gamma,
         beta,
         bias,
-        epsilon,
-        stash_type,
+        layer_norm,
         **_,
     ):
+        epsilon = layer_norm.producer().attributes.get_float("epsilon")
```

Comment:
I think you should use `get_float("epsilon", 1e-5)` to use the default value if it is not explicitly specified.